google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -266,8 +363,12 @@ int main(int argc, char* argv[]) try {
   google::cloud::bigtable::InstanceAdmin instance_admin(instance_admin_client);
   //! [connect instance admin]
 
-  if (command == "create-instance") {
+  if (command == "run") {
```

Comment:
Consider "run-tutorial" or "run-full-example" for the name of the command.

Reply:
The node.js examples uses `node $0 run --instance [instanceName] --cluster [clusterName]` as the command.  It would be ideal to keep the command names name across samples.  We should get Jisha's guidance for this type of change.